content/docs/user-guide/managing-external-data.md

```diff
@@ -65,7 +65,7 @@ location, e.g. `s3://mybucket/cache` or `ssh://user@example.com/cache` (see the
 examples below).
 
 > ⚠️ An external cache could be
-> [shared](/doc/use-cases/shared-development-server) among copies of a DVC
+> [shared](/doc/use-cases/sharing-resources-efficiently) among copies of a DVC
```

Comment:
same here - and check other place please

Reply:
Oops. All fixed now, my bad. That was sloppy 😓